src/main/js/SaveApi.js

```diff
@@ -6,22 +6,23 @@ import { Fetch, getRestUrl, sseService, loadingIndicator, capabilityAugmenter }
 export class SaveApi {
 
     indexRepo(organization, teamName, repoName) {
-        const createUrl = `${getRestUrl({organization})}/pipelines/`;
-
+        const createUrl = repoName ? `${getRestUrl({organization})}pipelines/` : `${getRestUrl({organization, pipeline: teamName})}`;
```

Comment:
Because this is the right place to trigger indexing. It's already nicely wrapped inside `RunApi.startRun()`, creating another wrapper over it doesn't do much, but feel free to refactor or wrap with something else.